instrumentation/spring/spring-webmvc-3.1/javaagent/src/test/groovy/test/filter/ServletFilterTest.groovy

```diff
@@ -73,33 +74,20 @@ class ServletFilterTest extends HttpServerTest<ConfigurableApplicationContext> i
   }
 
   @Override
-  boolean testNotFound() {
-    // FIXME: the instrumentation adds an extra controller span which is not consistent.
-    // Fix tests or remove extra span.
-    false
-  }
-
-  @Override
-  void handlerSpan(TraceAssert trace, int index, Object parent, String method = "GET", ServerEndpoint endpoint = SUCCESS) {
+  void handlerSpan(TraceAssert trace, int index, Object parent, String method = "GET", ServerEndpoint endpoint) {
     trace.span(index) {
-      name "TestController.${endpoint.name().toLowerCase()}"
+      name "ResourceHttpRequestHandler.handleRequest"
       kind INTERNAL
-      errored endpoint == EXCEPTION
       childOf((SpanData) parent)
-      if (endpoint == EXCEPTION) {
-        errorEvent(Exception, EXCEPTION.body)
-      }
     }
   }
```

Comment:
this `handlerSpan` method was previously unused since `hasHandlerSpan()` previously always returned false above